corehq/apps/toggle_ui/tests/test_toggles.py

```diff
@@ -16,13 +16,14 @@
     DynamicallyPredictablyRandomToggle,
     NAMESPACE_EMAIL_DOMAIN,
 )
-from toggle.shortcuts import (
+from corehq.apps.toggle_ui.shortcuts import (
     namespaced_item,
     find_users_with_toggle_enabled,
     find_domains_with_toggle_enabled,
+    toggle_enabled,
+    set_toggle,
 )
-from .models import generate_toggle_id, Toggle
-from .shortcuts import toggle_enabled, set_toggle
+from corehq.apps.toggle_ui.models import generate_toggle_id, Toggle
```

Comment:
nit: I like relative imports in tests when importing the things being tested, which are typically nearby. I think it would have worked to add an extra dot to the two import lines that were removed above, and also merge the `toggle.shortcuts` one in with the `..shortcuts` one.